### PR TITLE
Feat/add developer

### PR DESCRIPTION
**Pull request by rohit89gkmit**
removed duplicate script file

### CODE REVIEW

File: src/styles/style.css
Comment (shelly19gkmit) [inline src/styles/style.css:1]: Do not commit empty files

**Comment by rohit89gkmit** [comment]
fixed. please check

*Rohit Sharma*Development Intern

[image: Photo] <https://www.gkmit.co/>
E:  ***@***.***
P:  +91 8955675971 <+918955675971>
A:  A-SQUARE, 2nd floor, 100 Feet Rd, Shobhagpura
<https://goo.gl/maps/YLPZ1TGAXTL72J3S6>, Udaipur, Rajasthan 313011
<https://goo.gl/maps/YLPZ1TGAXTL72J3S6>

[image: facebook icon] <https://www.facebook.com/gkmit.co>  [image: twitter
icon] <https://twitter.com/i/flow/login?redirect_after_login=%2FGKMIT1>
 [image:
linkedin icon] <https://www.linkedin.com/company/gkmit-pvt-ltd/>

Disclaimer: This email and its attachments may contain privileged and
confidential information. If you are not an intended recipient you are
requested not to reproduce, copy, disseminate or in any manner distribute
this email communication as the same is strictly prohibited. If you have
received this email in error, please notify the sender immediately by
return e-mail and delete the communication sent in error. Email
communications cannot be guaranteed to be secure and error-free and GKM IT
Pvt. Ltd. is not liable for any errors in the email communication or for
the proper, timely and complete transmission thereof.

File: src/styles/style.css
Comment (shelly19gkmit) [reply src/styles/style.css:1]: @rohit89gkmit ^^

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+package-lock.json
+dist
+style.css
```
Comment (adiTrivsme) [inline .gitignore]: do not add to `.gitignore` simply don't commit this file

File: src/styles/style.css
Comment (adiTrivsme) [reply src/styles/style.css:1]: @shelly19gkmit lets keep this for now (to save time), he will anyway add the styles in the next commit

**Review by adiTrivsme — APPROVED** [review]

**Pull request closed without merging**